In [4]:
#import url 
import requests

url='https://lefooding.com/fr/search?type=restaurant&filters%5Bgeo%5D%5Bcity%5D%5B%5D=526ba3f07f55b7f39a007e28&page=1&sort=&keyword=restaurant&per_page=10&total_pages=14'
r=requests.get(url)
print(url, r.status_code)


https://lefooding.com/fr/search?type=restaurant&filters%5Bgeo%5D%5Bcity%5D%5B%5D=526ba3f07f55b7f39a007e28&page=1&sort=&keyword=restaurant&per_page=10&total_pages=14 200


In [220]:
#on insère dans json pour pouvoir l'utiliser facilement
a = r.json()
print(a)

{'pagination': {'current_page': 1, 'total': 134, 'total_pages': 14, 'per_page': 10}, 'facets': [{'type': 'price', 'facets': [{'id': '526dac737f55b742d5002065', 'count': 65}, {'id': '526dac747f55b742d50020d3', 'count': 42}, {'id': '526dac747f55b742d50020d5', 'count': 16}, {'id': '526dac727f55b742d5002063', 'count': 11}]}, {'type': 'essentials', 'facets': [{'id': '526dac727f55b742d5002053', 'count': 80}, {'id': '526dac727f55b742d5002055', 'count': 22}, {'id': '526dac727f55b742d5002051', 'count': 16}]}, {'type': 'zeplus', 'facets': [{'id': '526dac727f55b742d500203f', 'count': 99}, {'id': '526dac727f55b742d5002037', 'count': 78}, {'id': '529707fc42719aba96000001', 'count': 74}, {'id': '526dac727f55b742d5002035', 'count': 61}, {'id': '526dac727f55b742d5002045', 'count': 55}, {'id': '526dac727f55b742d500203b', 'count': 33}, {'id': '526dac727f55b742d500204d', 'count': 20}, {'id': '526dac727f55b742d5002041', 'count': 19}, {'id': '526dac727f55b742d5002049', 'count': 15}, {'id': '526dac727f55b74

In [221]:
#on crée un fichier projet en txt
my_data_file = open('projet.txt', 'w')

import csv
import json

x="""[
{
'pagination': {'current_page': 1,
  'total': 134,
  'total_pages': 14,
  'per_page': 10},
 'facets': [{'type': 'price',
   'facets': [{'id': '526dac737f55b742d5002065', 'count': 65},
    {'id': '526dac747f55b742d50020d3', 'count': 42},
    {'id': '526dac747f55b742d50020d5', 'count': 16},
    {'id': '526dac727f55b742d5002063', 'count': 11}]},
  {'type': 'essentials',
   'facets': [{'id': '526dac727f55b742d5002053', 'count': 80},
    {'id': '526dac727f55b742d5002055', 'count': 22},
    {'id': '526dac727f55b742d5002051', 'count': 16}]},
  {'type': 'zeplus',
   'facets': [{'id': '526dac727f55b742d500203f', 'count': 99},
    {'id': '526dac727f55b742d5002037', 'count': 78},
    {'id': '529707fc42719aba96000001', 'count': 74},
    {'id': '526dac727f55b742d5002035', 'count': 61},
    {'id': '526dac727f55b742d5002045', 'count': 55},
    {'id': '526dac727f55b742d500203b', 'count': 33},
    {'id': '526dac727f55b742d500204d', 'count': 20},
    {'id': '526dac727f55b742d5002041', 'count': 19},
    {'id': '526dac727f55b742d5002049', 'count': 15},
    {'id': '526dac727f55b742d500204b', 'count': 11},
    {'id': '526dac727f55b742d500203d', 'count': 10},
    {'id': '526dac727f55b742d5002043', 'count': 8}]}],
 'results': [{'id': '5c8fba9a42719a5b5100003e',
   'type': 'Restaurant',
   'name': 'Restaurant du Badaboum',
   'address': {'_id': '5c8fba9a42719a5b5100003c',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2019-03-18T16:34:51+01:00',
    'department': 'Paris',
    'position': [48.85345299999999, 2.3757510000000366],
    'postal_code': '75011',
    'region': 'Région Île-de-France',
    'street_name_1': '2, rue des Taillandiers',
    'street_name_2': '',
    'updated_at': '2019-03-18T16:34:51+01:00'},
   'phone_num': '+33 1 85 09 75 46',
   'cellphone_num': '',
   'cravings': ["Cuisine d'auteur",
    'Cuisine du marché',
    'Néobistrot',
    'Bar à vins / Cave à manger'],
   'extras': ['Antidépresseur', 'Terrasse', '13 à table', 'Manger seul '],
   'price': ['De 16 à 35 €'],
   'essentials': ['Feeling'],
   'url': 'https://lefooding.com/fr/restaurants/restaurant-du-badaboum-paris',
   'tags': [{'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Antidépresseur'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Terrasse'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': '13 à table'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Feeling'},
    {'category': 'Prix', 'type': 'price', 'name': 'De 16 à 35 €'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': "Cuisine d'auteur"},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Cuisine du marché'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Néobistrot'},
    {'category': 'Cuisine',
     'type': 'kitchen',
     'name': 'Bar à vins / Cave à manger'},
    {'category': 'Transport', 'type': 'transport', 'name': 'Ledru Rollin'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Manger seul '}],
   'sorted_tags': {'zeplus': ['Antidépresseur',
     'Terrasse',
     '13 à table',
     'Manger seul '],
    'essentials': ['Feeling'],
    'price': ['De 16 à 35 €'],
    'kitchen': ["Cuisine d'auteur",
     'Cuisine du marché',
     'Néobistrot',
     'Bar à vins / Cave à manger'],
    'transport': ['Ledru Rollin']},
   'website': 'www.badaboum.paris',
   'thumbnail': '/media/W1siZiIsIjIwMTkvMDMvMTgvMTZfMzRfNTBfNDU3X0JBREFCT1VNXzRfMi5qcGVnIl0sWyJwIiwidGh1bWIiLCIyMDd4MTM2IyJdXQ/BADABOUM%204%202.jpeg?sha=96406633'},
  {'id': '5a28125f42719a899e000001',
   'type': 'Restaurant',
   'name': 'Restaurant Passerini',
   'address': {'_id': '573a477d42719afdc000000d',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2016-05-17T00:19:41+02:00',
    'department': 'Paris',
    'position': [48.8494184, 2.374820600000021],
    'postal_code': '75012',
    'region': 'Région Île-de-France',
    'street_name_1': '65, rue Traversière',
    'street_name_2': '',
    'updated_at': '2018-12-05T11:17:35+01:00'},
   'phone_num': '+ 33 1 43 42 27 56',
   'cellphone_num': '',
   'cravings': ["Cuisine d'auteur", 'Italien', 'Néobistrot'],
   'extras': ['Antidépresseur', 'Terrasse'],
   'price': ['De 36 à 50 €'],
   'essentials': ['Trop Bon', 'Palmarès'],
   'url': 'https://lefooding.com/fr/restaurants/restaurant-passerini-paris-3',
   'tags': [{'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Antidépresseur'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Terrasse'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Trop Bon'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Palmarès'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': "Cuisine d'auteur"},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Italien'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Néobistrot'},
    {'category': 'Prix', 'type': 'price', 'name': 'De 36 à 50 €'},
    {'category': 'Transport', 'type': 'transport', 'name': 'Ledru Rollin'}],
   'sorted_tags': {'zeplus': ['Antidépresseur', 'Terrasse'],
    'essentials': ['Trop Bon', 'Palmarès'],
    'kitchen': ["Cuisine d'auteur", 'Italien', 'Néobistrot'],
    'price': ['De 36 à 50 €'],
    'transport': ['Ledru Rollin']},
   'website': 'www.passerini.paris',
   'thumbnail': '/media/W1siZiIsIjIwMTYvMDYvMjQvMThfMTJfMzRfODg0X3Jlc3RhdXJhbnRfcGFzc2VyaW5pX3BhcmlzLmpwZyJdLFsicCIsInRodW1iIiwiMjA3eDEzNiMiXV0/restaurant-passerini-paris.jpg?sha=ebd4370d'},
  {'id': '5a26861942719aec84000001',
   'type': 'Restaurant',
   'name': 'A.T. Restaurant ',
   'address': {'_id': '584581ee42719ae1dd000038',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2016-12-05T16:04:14+01:00',
    'department': 'Paris',
    'position': [48.84974, 2.3544939999999315],
    'postal_code': '75005',
    'region': 'Région Île-de-France',
    'street_name_1': '4 bis, rue du Cardinal-Lemoine',
    'street_name_2': '',
    'updated_at': '2018-11-28T17:21:51+01:00'},
   'phone_num': '+33 1 56 81 94 08 ',
   'cellphone_num': '',
   'cravings': ["Cuisine d'auteur", 'Néobistrot'],
   'extras': ['Ouvert le lundi', 'Fais-moi mal'],
   'price': ['Plus de 51 €'],
   'essentials': ['Trop Bon'],
   'url': 'https://lefooding.com/fr/restaurants/a-t-restaurant-paris',
   'tags': [{'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Ouvert le lundi'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Fais-moi mal'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Trop Bon'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': "Cuisine d'auteur"},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Néobistrot'},
    {'category': 'Prix', 'type': 'price', 'name': 'Plus de 51 €'},
    {'category': 'Transport',
     'type': 'transport',
     'name': 'Maubert - Mutualité'},
    {'category': 'Transport', 'type': 'transport', 'name': 'Cardinal Lemoine'},
    {'category': 'Transport', 'type': 'transport', 'name': 'Pont Marie'}],
   'sorted_tags': {'zeplus': ['Ouvert le lundi', 'Fais-moi mal'],
    'essentials': ['Trop Bon'],
    'kitchen': ["Cuisine d'auteur", 'Néobistrot'],
    'price': ['Plus de 51 €'],
    'transport': ['Maubert - Mutualité', 'Cardinal Lemoine', 'Pont Marie']},
   'website': 'www.atsushitanaka.com',
   'thumbnail': '/media/W1siZiIsIjIwMTYvMDEvMDcvMTBfNThfMzFfMjA5X3Jlc3RhdXJhbnRfQVRfcGFyaXNfLmpwZyJdLFsicCIsInRodW1iIiwiMjA3eDEzNiMiXV0/restaurant-AT-paris%20.jpg?sha=887ad505'},
  {'id': '5a4fab5142719a3ba6000019',
   'type': 'Restaurant',
   'name': 'Chameleon Restaurant',
   'address': {'_id': '5847d0c642719a18fc00000b',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2016-12-07T10:05:10+01:00',
    'department': 'Paris',
    'position': [48.86921700000001, 2.358015499999965],
    'postal_code': '75010',
    'region': 'Région Île-de-France',
    'street_name_1': '70, rue René-Boulanger ',
    'street_name_2': '',
    'updated_at': '2018-12-03T11:46:29+01:00'},
   'phone_num': '+33 1 42 08 99 41',
   'cellphone_num': '',
   'cravings': ["Cuisine d'auteur", 'Néobistrot'],
   'extras': ['Ouvert le lundi',
    'Antidépresseur',
    'Faim de nuit',
    'Terrasse',
    '13 à table'],
   'price': ['De 16 à 35 €'],
   'essentials': [],
   'url': 'https://lefooding.com/fr/restaurants/chameleon-restaurant-paris-3',
   'tags': [{'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Ouvert le lundi'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Antidépresseur'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Faim de nuit'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Terrasse'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': '13 à table'},
    {'category': 'Prix', 'type': 'price', 'name': 'De 16 à 35 €'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': "Cuisine d'auteur"},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Néobistrot'},
    {'category': 'Transport', 'type': 'transport', 'name': 'République'},
    {'category': 'Transport',
     'type': 'transport',
     'name': 'Jacques Bonsergent'},
    {'category': 'Transport',
     'type': 'transport',
     'name': 'Strasbourg-Saint-Denis'}],
   'sorted_tags': {'zeplus': ['Ouvert le lundi',
     'Antidépresseur',
     'Faim de nuit',
     'Terrasse',
     '13 à table'],
    'price': ['De 16 à 35 €'],
    'kitchen': ["Cuisine d'auteur", 'Néobistrot'],
    'transport': ['République',
     'Jacques Bonsergent',
     'Strasbourg-Saint-Denis']},
   'website': 'www.chameleonrestaurant.fr',
   'thumbnail': '/media/W1siZiIsIjIwMTYvMDIvMDIvMTVfMzJfMDlfODIzX3Jlc3RhdXJhbnRfY2hhbWVsZW9uX0NoYW1lbGVvbl9wYXJpcy5qcGciXSxbInAiLCJ0aHVtYiIsIjIwN3gxMzYjIl1d/restaurant-chameleon-%C2%A9Chameleon-paris.jpg?sha=5bb08f11'},
  {'id': '5a297a7442719a8acf000007',
   'type': 'Restaurant',
   'name': 'Restaurant David Toutain',
   'address': {'_id': '5845332e42719afd06000012',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2016-12-05T10:28:14+01:00',
    'department': 'Paris',
    'position': [48.8602571, 2.3097285000000056],
    'postal_code': '75007',
    'region': 'Région Île-de-France',
    'street_name_1': '29 Rue Surcouf',
    'street_name_2': '',
    'updated_at': '2018-11-29T14:10:52+01:00'},
   'phone_num': '+33 1 45 50 11 10',
   'cellphone_num': '',
   'cravings': ["Cuisine d'auteur", 'Néobistrot'],
   'extras': ['Ouvert le lundi', 'Fais-moi mal'],
   'price': ['Plus de 51 €'],
   'essentials': ['Trop Bon', 'Palmarès'],
   'url': 'https://lefooding.com/fr/restaurants/restaurant-david-toutain-paris-5',
   'tags': [{'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Ouvert le lundi'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Fais-moi mal'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Trop Bon'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Palmarès'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': "Cuisine d'auteur"},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Néobistrot'},
    {'category': 'Prix', 'type': 'price', 'name': 'Plus de 51 €'},
    {'category': 'Transport',
     'type': 'transport',
     'name': 'La Tour-Maubourg'}],
   'sorted_tags': {'zeplus': ['Ouvert le lundi', 'Fais-moi mal'],
    'essentials': ['Trop Bon', 'Palmarès'],
    'kitchen': ["Cuisine d'auteur", 'Néobistrot'],
    'price': ['Plus de 51 €'],
    'transport': ['La Tour-Maubourg']},
   'website': 'https://www.davidtoutain.com',
   'thumbnail': '/media/W1siZiIsIjIwMTQvMDEvMDYvMTdfMjlfMzJfNzkxX3Jlc3RhdXJhbnRfZGF2aWRfdG91dGFpbl9wYXJpcy5qcGciXSxbInAiLCJ0aHVtYiIsIjIwN3gxMzYjIl1d/restaurant-david-toutain-paris.jpg?sha=14c4a0b3'},
  {'id': '5c04f34742719ad8bf000370',
   'type': 'Restaurant',
   'name': 'Barak',
   'address': {'_id': '5c04f34742719ad8bf00036f',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2018-12-03T10:11:36+01:00',
    'department': 'Paris',
    'position': [48.87551819999999, 2.3482715999999755],
    'postal_code': '75010',
    'region': 'Région Île-de-France',
    'street_name_1': '62, rue du Faubourg-Poissonnière',
    'street_name_2': '',
    'updated_at': '2018-12-03T10:11:36+01:00'},
   'phone_num': 'Pas de téléphone ',
   'cellphone_num': '',
   'cravings': ['Kebabs', 'Sandwichs / Bagels', 'Lèche-doigts'],
   'extras': ['Ouvert le dimanche',
    'Ouvert le lundi',
    'Take-away',
    'Kid friendly',
    'Terrasse',
    '13 à table',
    'Manger seul '],
   'price': ['Moins de 15 €'],
   'essentials': [],
   'url': 'https://lefooding.com/fr/restaurants/restaurant-barak-paris',
   'tags': [{'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Ouvert le dimanche'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Ouvert le lundi'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Take-away'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Kid friendly'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Terrasse'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': '13 à table'},
    {'category': 'Prix', 'type': 'price', 'name': 'Moins de 15 €'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Kebabs'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Sandwichs / Bagels'},
    {'category': 'Transport', 'type': 'transport', 'name': 'Poissonnière'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Manger seul '},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Lèche-doigts'}],
   'sorted_tags': {'zeplus': ['Ouvert le dimanche',
     'Ouvert le lundi',
     'Take-away',
     'Kid friendly',
     'Terrasse',
     '13 à table',
     'Manger seul '],
    'price': ['Moins de 15 €'],
    'kitchen': ['Kebabs', 'Sandwichs / Bagels', 'Lèche-doigts'],
    'transport': ['Poissonnière']},
   'website': 'barak.fr',
   'thumbnail': '/media/W1siZiIsIjIwMTkvMTEvMjAvMTRfMzVfMjVfNDYxX3Jlc3RhdXJhbnRfYmFyYWtfcGFyaXMuanBnIl0sWyJwIiwidGh1bWIiLCIyMDd4MTM2IyJdXQ/restaurant-barak-paris.jpg?sha=c9af1574'},
  {'id': '5a267bef42719a52bb00000f',
   'type': 'Restaurant',
   'name': 'Le Mary Celeste',
   'address': {'_id': '5a267bef42719a52bb000015',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2017-12-05T11:58:55+01:00',
    'department': 'Paris',
    'position': [48.861688, 2.364839599999982],
    'postal_code': '75003',
    'region': 'Région Île-de-France',
    'street_name_1': '1, rue Commines',
    'street_name_2': '',
    'updated_at': '2018-11-28T12:03:05+01:00'},
   'phone_num': '+33 1 42 77 98 37',
   'cellphone_num': '',
   'cravings': ['Bar à vins / Cave à manger'],
   'extras': ['Ouvert le dimanche',
    'Ouvert le lundi',
    'Antidépresseur',
    'Faim de nuit',
    'Terrasse',
    'Voir et se faire voir'],
   'price': ['De 16 à 35 €'],
   'essentials': ['Feeling', 'Palmarès'],
   'url': 'https://lefooding.com/fr/restaurants/restaurant-le-mary-celeste-paris',
   'tags': [{'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Ouvert le dimanche'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Ouvert le lundi'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Antidépresseur'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Faim de nuit'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Terrasse'},
    {'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Voir et se faire voir'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Feeling'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Palmarès'},
    {'category': 'Prix', 'type': 'price', 'name': 'De 16 à 35 €'},
    {'category': 'Cuisine',
     'type': 'kitchen',
     'name': 'Bar à vins / Cave à manger'},
    {'category': 'Transport',
     'type': 'transport',
     'name': 'Filles du Calvaire'},
    {'category': 'Transport',
     'type': 'transport',
     'name': 'Saint-Sébastien-Froissart'}],
   'sorted_tags': {'zeplus': ['Ouvert le dimanche',
     'Ouvert le lundi',
     'Antidépresseur',
     'Faim de nuit',
     'Terrasse',
     'Voir et se faire voir'],
    'essentials': ['Feeling', 'Palmarès'],
    'price': ['De 16 à 35 €'],
    'kitchen': ['Bar à vins / Cave à manger'],
    'transport': ['Filles du Calvaire', 'Saint-Sébastien-Froissart']},
   'website': 'lemaryceleste.com',
   'thumbnail': '/media/W1siZiIsIjIwMTQvMDQvMjIvMTdfMjZfNThfNzQyX21hcnljZWxlc3RlLmpwZyJdLFsicCIsInRodW1iIiwiMjA3eDEzNiMiXV0/maryceleste.jpg?sha=c9259b1b'},
  {'id': '5c62fbcf42719a9ac9000054',
   'type': 'Restaurant',
   'name': 'Early June',
   'address': {'_id': '5c62fbcf42719a9ac9000052',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2019-02-12T18:01:04+01:00',
    'department': 'Paris',
    'position': [48.87291459999999, 2.363441200000011],
    'postal_code': '75010',
    'region': 'Région Île-de-France',
    'street_name_1': '19, rue Jean Poulmarch',
    'street_name_2': '',
    'updated_at': '2019-12-19T15:32:12+01:00'},
   'phone_num': '+33142854074',
   'cellphone_num': '',
   'cravings': ['Bar à vins / Cave à manger'],
   'extras': ['Take-away', 'Antidépresseur', 'Terrasse'],
   'price': ['De 16 à 35 €'],
   'essentials': ['Feeling'],
   'url': 'https://lefooding.com/fr/restaurants/restaurant-early-june-paris',
   'tags': [{'category': 'Les plus', 'type': 'zeplus', 'name': 'Take-away'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Antidépresseur'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Terrasse'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Feeling'},
    {'category': 'Prix', 'type': 'price', 'name': 'De 16 à 35 €'},
    {'category': 'Cuisine',
     'type': 'kitchen',
     'name': 'Bar à vins / Cave à manger'},
    {'category': 'Transport',
     'type': 'transport',
     'name': 'Jacques Bonsergent'},
    {'category': 'Transport', 'type': 'transport', 'name': "Gare de l'Est"}],
   'sorted_tags': {'zeplus': ['Take-away', 'Antidépresseur', 'Terrasse'],
    'essentials': ['Feeling'],
    'price': ['De 16 à 35 €'],
    'kitchen': ['Bar à vins / Cave à manger'],
    'transport': ['Jacques Bonsergent', "Gare de l'Est"]},
   'website': 'www.early-june.fr',
   'thumbnail': '/media/W1siZiIsIjIwMTkvMDcvMjYvMTVfMTZfMTZfOTk0X3Jlc3RhdXJhbnRfZWFybHlfanVuZV9wYXJpcy5KUEciXSxbInAiLCJ0aHVtYiIsIjIwN3gxMzYjIl1d/restaurant-early-june-paris.JPG?sha=e803a590'},
  {'id': '5cec0f2542719aa67d0003ea',
   'type': 'Restaurant',
   'name': 'Adar',
   'address': {'_id': '5cec0f2542719aa67d0003e8',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2019-05-27T18:24:05+02:00',
    'department': 'Paris',
    'position': [48.8709479, 2.341645900000003],
    'postal_code': '75002',
    'region': 'Région Île-de-France',
    'street_name_1': '49, passage des Panoramas',
    'street_name_2': '',
    'updated_at': '2019-05-27T18:24:05+02:00'},
   'phone_num': '',
   'cellphone_num': '+33 6 64 49 18 68 ',
   'cravings': ['Cuisine du marché',
    'Méditerranéen',
    'Sandwichs / Bagels',
    'Lèche-doigts',
    'Israélien'],
   'extras': ['Ouvert le lundi',
    'Take-away',
    'Antidépresseur',
    'Terrasse',
    'Manger seul '],
   'price': ['De 16 à 35 €'],
   'essentials': ['Feeling'],
   'url': 'https://lefooding.com/fr/restaurants/restaurant-adar-paris',
   'tags': [{'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Ouvert le lundi'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Take-away'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Antidépresseur'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Terrasse'},
    {'category': 'Les essentiels', 'type': 'essentials', 'name': 'Feeling'},
    {'category': 'Prix', 'type': 'price', 'name': 'De 16 à 35 €'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Cuisine du marché'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Méditerranéen'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Sandwichs / Bagels'},
    {'category': 'Transport',
     'type': 'transport',
     'name': 'Grands Boulevards'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Manger seul '},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Lèche-doigts'},
    {'category': 'Cuisine', 'type': 'kitchen', 'name': 'Israélien'}],
   'sorted_tags': {'zeplus': ['Ouvert le lundi',
     'Take-away',
     'Antidépresseur',
     'Terrasse',
     'Manger seul '],
    'essentials': ['Feeling'],
    'price': ['De 16 à 35 €'],
    'kitchen': ['Cuisine du marché',
     'Méditerranéen',
     'Sandwichs / Bagels',
     'Lèche-doigts',
     'Israélien'],
    'transport': ['Grands Boulevards']},
   'website': 'www.adar-paris.com',
   'thumbnail': '/media/W1siZiIsIjIwMTkvMDUvMjcvMThfMjRfMDVfMjgzX3Jlc3RhdXJhbnRfYWRhcl9wYXJpcy5qcGVnIl0sWyJwIiwidGh1bWIiLCIyMDd4MTM2IyJdXQ/restaurant-adar-paris.jpeg?sha=8ca1ed97'},
  {'id': '5b87fc3942719a75a000006d',
   'type': 'Restaurant',
   'name': 'Fulgurances, en face',
   'address': {'_id': '5b87fc3942719a75a000006c',
    'city': 'Paris',
    'country': 'France',
    'created_at': '2018-08-30T16:16:26+02:00',
    'department': 'Paris',
    'position': [48.8529714, 2.389381299999968],
    'postal_code': '75011',
    'region': 'Région Île-de-France',
    'street_name_1': '5, rue Alexandre-Dumas',
    'street_name_2': '',
    'updated_at': '2018-08-30T16:16:26+02:00'},
   'phone_num': '+33 1 45 68 92 52',
   'cellphone_num': '',
   'cravings': ['Planches / Assiettes froides', 'Bar à vins / Cave à manger'],
   'extras': ['Ouvert le lundi',
    'Take-away',
    'Antidépresseur',
    'Faim de nuit',
    'Terrasse',
    'Manger seul '],
   'price': ['De 16 à 35 €'],
   'essentials': [],
   'url': 'https://lefooding.com/fr/restaurants/restaurant-fulgurances-en-face-paris',
   'tags': [{'category': 'Les plus',
     'type': 'zeplus',
     'name': 'Ouvert le lundi'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Take-away'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Antidépresseur'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Faim de nuit'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Terrasse'},
    {'category': 'Prix', 'type': 'price', 'name': 'De 16 à 35 €'},
    {'category': 'Cuisine',
     'type': 'kitchen',
     'name': 'Planches / Assiettes froides'},
    {'category': 'Cuisine',
     'type': 'kitchen',
     'name': 'Bar à vins / Cave à manger'},
    {'category': 'Transport', 'type': 'transport', 'name': 'Alexandre Dumas'},
    {'category': 'Transport', 'type': 'transport', 'name': 'Charonne'},
    {'category': 'Les plus', 'type': 'zeplus', 'name': 'Manger seul '}],
   'sorted_tags': {'zeplus': ['Ouvert le lundi',
     'Take-away',
     'Antidépresseur',
     'Faim de nuit',
     'Terrasse',
     'Manger seul '],
    'price': ['De 16 à 35 €'],
    'kitchen': ['Planches / Assiettes froides', 'Bar à vins / Cave à manger'],
    'transport': ['Alexandre Dumas', 'Charonne']},
   'website': 'fulgurances.com',
   'thumbnail': '/media/W1siZiIsIjIwMTgvMDkvMTgvMTFfMzZfNTFfNjUzX3Jlc3RhdXJhbnRfZnVsZ3VyYW5jZXNfZW5fZmFjZV9wYXJpc18xLmpwZWciXSxbInAiLCJ0aHVtYiIsIjIwN3gxMzYjIl1d/restaurant-fulgurances-en-face-paris-1.jpeg?sha=da2d0dfa'}],
 'preselected_filters': {'geo': [{'type': 'city',
    'id': '526ba3f07f55b7f39a007e28',
    'name': 'Paris',
    'code': '75',
    'selected': True}]}}
    ]"""
    x =json.loards(x)

In [222]:
#on crée un fichier csv nommé projet csv
import csv
with open ('projet.csv','w') as csv_file:
    csv_writer=csv.writer(csv_file, delimiter=',')
    csv_writer.writerow(r)

In [229]:
#on range le projet csv dans panda
import pandas as pd
df =pandas.read_csv('projet.csv', sep='\t')
df = pd.json_normalize(a['results'])
df

id        type                      name  \
0  5c8fba9a42719a5b5100003e  Restaurant    Restaurant du Badaboum   
1  5a28125f42719a899e000001  Restaurant      Restaurant Passerini   
2  5a26861942719aec84000001  Restaurant          A.T. Restaurant    
3  5a4fab5142719a3ba6000019  Restaurant      Chameleon Restaurant   
4  5a297a7442719a8acf000007  Restaurant  Restaurant David Toutain   
5  5c04f34742719ad8bf000370  Restaurant                     Barak   
6  5a267bef42719a52bb00000f  Restaurant           Le Mary Celeste   
7  5c62fbcf42719a9ac9000054  Restaurant                Early June   
8  5cec0f2542719aa67d0003ea  Restaurant                      Adar   
9  5b87fc3942719a75a000006d  Restaurant      Fulgurances, en face   

            phone_num       cellphone_num  \
0   +33 1 85 09 75 46                       
1  + 33 1 43 42 27 56                       
2  +33 1 56 81 94 08                        
3   +33 1 42 08 99 41                       
4   +33 1 45 50 11 10                       
5   Pas de téléphone                        
6   +33 1 42 77 98 37                       
7        +33142854074                       
8                      +33 6 64 49 18 68    
9   +33 1 45 68 92 52                       

                                            cravings  \
0  [Cuisine d'auteur, Cuisine du marché, Néobistr...   
1            [Cuisine d'auteur, Italien, Néobistrot]   
2                     [Cuisine d'auteur, Néobistrot]   
3                     [Cuisine d'auteur, Néobistrot]   
4                     [Cuisine d'auteur, Néobistrot]   
5         [Kebabs, Sandwichs / Bagels, Lèche-doigts]   
6                       [Bar à vins / Cave à manger]   
7                       [Bar à vins / Cave à manger]   
8  [Cuisine du marché, Méditerranéen, Sandwichs /...   
9  [Planches / Assiettes froides, Bar à vins / Ca...   

                                              extras            price  \
0  [Antidépresseur, Terrasse, 13 à table, Manger ...   [De 16 à 35 €]   
1                         [Antidépresseur, Terrasse]   [De 36 à 50 €]   
2                    [Ouvert le lundi, Fais-moi mal]   [Plus de 51 €]   
3  [Ouvert le lundi, Antidépresseur, Faim de nuit...   [De 16 à 35 €]   
4                    [Ouvert le lundi, Fais-moi mal]   [Plus de 51 €]   
5  [Ouvert le dimanche, Ouvert le lundi, Take-awa...  [Moins de 15 €]   
6  [Ouvert le dimanche, Ouvert le lundi, Antidépr...   [De 16 à 35 €]   
7              [Take-away, Antidépresseur, Terrasse]   [De 16 à 35 €]   
8  [Ouvert le lundi, Take-away, Antidépresseur, T...   [De 16 à 35 €]   
9  [Ouvert le lundi, Take-away, Antidépresseur, F...   [De 16 à 35 €]   

             essentials                                                url  \
0             [Feeling]  https://lefooding.com/fr/restaurants/restauran...   
1  [Trop Bon, Palmarès]  https://lefooding.com/fr/restaurants/restauran...   
2            [Trop Bon]  https://lefooding.com/fr/restaurants/a-t-resta...   
3                    []  https://lefooding.com/fr/restaurants/chameleon...   
4  [Trop Bon, Palmarès]  https://lefooding.com/fr/restaurants/restauran...   
5                    []  https://lefooding.com/fr/restaurants/restauran...   
6   [Feeling, Palmarès]  https://lefooding.com/fr/restaurants/restauran...   
7             [Feeling]  https://lefooding.com/fr/restaurants/restauran...   
8             [Feeling]  https://lefooding.com/fr/restaurants/restauran...   
9                    []  https://lefooding.com/fr/restaurants/restauran...   

   ... address.postal_code        address.region  \
0  ...               75011  Région Île-de-France   
1  ...               75012  Région Île-de-France   
2  ...               75005  Région Île-de-France   
3  ...               75010  Région Île-de-France   
4  ...               75007  Région Île-de-France   
5  ...               75010  Région Île-de-France   
6  ...               75003  Région Île-de-France   
7  ...               75010  Région Île-de-France   
8  ...               

In [201]:
# Je detruis la colonne ["url","address.region","address.street_name_2","address.updated_at","sorted_tags.essentials","type","cellphone_num","extras","essentials"]
sup = ["address.updated_at","sorted_tags.zeplus","sorted_tags.essentials","sorted_tags.price","sorted_tags.kitchen","sorted_tags.transport"]
df.drop(sup,axis=1,inplace=True)

KeyError: "['address.updated_at' 'sorted_tags.zeplus' 'sorted_tags.essentials'\n 'sorted_tags.price' 'sorted_tags.kitchen' 'sorted_tags.transport'] not found in axis"

In [235]:
#on crée la fonction premier pour séparer les cravings
def premier(liste):
    separator=','
    try:
        return separator.join(liste)
    except:
        return''


In [236]:
df['cravings']=df['cravings'].apply(premier)

In [241]:
df

id                      name           phone_num  \
0  5c8fba9a42719a5b5100003e    Restaurant du Badaboum   +33 1 85 09 75 46   
1  5a28125f42719a899e000001      Restaurant Passerini  + 33 1 43 42 27 56   
2  5a26861942719aec84000001          A.T. Restaurant   +33 1 56 81 94 08    
3  5a4fab5142719a3ba6000019      Chameleon Restaurant   +33 1 42 08 99 41   
4  5a297a7442719a8acf000007  Restaurant David Toutain   +33 1 45 50 11 10   
5  5c04f34742719ad8bf000370                     Barak   Pas de téléphone    
6  5a267bef42719a52bb00000f           Le Mary Celeste   +33 1 42 77 98 37   
7  5c62fbcf42719a9ac9000054                Early June        +33142854074   
8  5cec0f2542719aa67d0003ea                      Adar                       
9  5b87fc3942719a75a000006d      Fulgurances, en face   +33 1 45 68 92 52   

                                            cravings            price  \
0  Cuisine d'auteur,Cuisine du marché,Néobistrot,...   [De 16 à 35 €]   
1                Cuisine d'auteur,Italien,Néobistrot   [De 36 à 50 €]   
2                        Cuisine d'auteur,Néobistrot   [Plus de 51 €]   
3                        Cuisine d'auteur,Néobistrot   [De 16 à 35 €]   
4                        Cuisine d'auteur,Néobistrot   [Plus de 51 €]   
5             Kebabs,Sandwichs / Bagels,Lèche-doigts  [Moins de 15 €]   
6                         Bar à vins / Cave à manger   [De 16 à 35 €]   
7                         Bar à vins / Cave à manger   [De 16 à 35 €]   
8  Cuisine du marché,Méditerranéen,Sandwichs / Ba...   [De 16 à 35 €]   
9  Planches / Assiettes froides,Bar à vins / Cave...   [De 16 à 35 €]   

                        website  \
0            www.badaboum.paris   
1           www.passerini.paris   
2         www.atsushitanaka.com   
3    www.chameleonrestaurant.fr   
4  https://www.davidtoutain.com   
5                      barak.fr   
6             lemaryceleste.com   
7             www.early-june.fr   
8            www.adar-paris.com   
9               fulgurances.com   

                                           thumbnail  \
0  /media/W1siZiIsIjIwMTkvMDMvMTgvMTZfMzRfNTBfNDU...   
1  /media/W1siZiIsIjIwMTYvMDYvMjQvMThfMTJfMzRfODg...   
2  /media/W1siZiIsIjIwMTYvMDEvMDcvMTBfNThfMzFfMjA...   
3  /media/W1siZiIsIjIwMTYvMDIvMDIvMTVfMzJfMDlfODI...   
4  /media/W1siZiIsIjIwMTQvMDEvMDYvMTdfMjlfMzJfNzk...   
5  /media/W1siZiIsIjIwMTkvMTEvMjAvMTRfMzVfMjVfNDY...   
6  /media/W1siZiIsIjIwMTQvMDQvMjIvMTdfMjZfNThfNzQ...   
7  /media/W1siZiIsIjIwMTkvMDcvMjYvMTVfMTZfMTZfOTk...   
8  /media/W1siZiIsIjIwMTkvMDUvMjcvMThfMjRfMDVfMjg...   
9  /media/W1siZiIsIjIwMTgvMDkvMTgvMTFfMzZfNTFfNjU...   

                address._id address.city address.country  \
0  5c8fba9a42719a5b5100003c        Paris          France   
1  573a477d42719afdc000000d        Paris          France   
2  584581ee42719ae1dd000038        Paris          France   
3  5847d0c642719a18fc00000b        Paris          France   
4  5845332e42719afd06000012        Paris          France   
5  5c04f34742719ad8bf00036f        Paris          France   
6  5a267bef42719a52bb000015        Paris          France   
7  5c62fbcf42719a9ac9000052        Paris          France   
8  5cec0f2542719aa67d0003e8        Paris          France   
9  5b87fc3942719a75a000006c        Paris          France   

          address.created_at address.department  \
0  2019-03-18T16:34:51+01:00              Paris   
1  2016-05-17T00:19:41+02:00              Paris   
2  2016-12-05T16:04:14+01:00              Paris   
3  2016-12-07T10:05:10+01:00              Paris   
4  2016-12-05T10:28:14+01:00              Paris   
5  2018-12-03T10:11:36+01:00              Paris   
6  2017-12-05T11:58:55+01:00              Paris   
7  2019-02-12T18:01:04+01:00              Paris   
8  2019-05-27T18:24:05+02:00              Paris   
9  2018-08-30T16:16:26+02:00              Paris   

                          address.position address.postal_code  \
0  [48.85345299999999, 2.3757510000000366]               75011   
1          [48.

In [243]:
#on test pour voir si la recherche en fonction du code postale et cravings
code_postal='75007'
spe_str="Cuisine d'auteur"
df[(df['address.postal_code']==code_postal) & (df['cravings'].str.contains(spe_str))]

id                      name          phone_num  \
4  5a297a7442719a8acf000007  Restaurant David Toutain  +33 1 45 50 11 10   

                      cravings           price                       website  \
4  Cuisine d'auteur,Néobistrot  [Plus de 51 €]  https://www.davidtoutain.com   

                                           thumbnail  \
4  /media/W1siZiIsIjIwMTQvMDEvMDYvMTdfMjlfMzJfNzk...   

                address._id address.city address.country  \
4  5845332e42719afd06000012        Paris          France   

          address.created_at address.department  \
4  2016-12-05T10:28:14+01:00              Paris   

                   address.position address.postal_code        address.region  \
4  [48.8602571, 2.3097285000000056]               75007  Région Île-de-France   

  address.street_name_1         address.updated_at  \
4        29 Rue Surcouf  2018-11-29T14:10:52+01:00   

                sorted_tags.zeplus sorted_tags.price  \
4  [Ouvert le lundi, Fais-moi mal]    [Plus de 51 €]   

              sorted_tags.kitchen  
4  [Cuisine d'auteur, Néobistrot]

In [244]:
# Je detruis les colonnes pour nettoyer 
sup = ["thumbnail","address._id","address.city","address.country","address.created_at","address.department","address.region","address.updated_at","sorted_tags.zeplus","sorted_tags.price","sorted_tags.kitchen"]
df.drop(sup,axis=1,inplace=True)

In [246]:
#je vérifie si c'est bien nettoyer 
df

id                      name           phone_num  \
0  5c8fba9a42719a5b5100003e    Restaurant du Badaboum   +33 1 85 09 75 46   
1  5a28125f42719a899e000001      Restaurant Passerini  + 33 1 43 42 27 56   
2  5a26861942719aec84000001          A.T. Restaurant   +33 1 56 81 94 08    
3  5a4fab5142719a3ba6000019      Chameleon Restaurant   +33 1 42 08 99 41   
4  5a297a7442719a8acf000007  Restaurant David Toutain   +33 1 45 50 11 10   
5  5c04f34742719ad8bf000370                     Barak   Pas de téléphone    
6  5a267bef42719a52bb00000f           Le Mary Celeste   +33 1 42 77 98 37   
7  5c62fbcf42719a9ac9000054                Early June        +33142854074   
8  5cec0f2542719aa67d0003ea                      Adar                       
9  5b87fc3942719a75a000006d      Fulgurances, en face   +33 1 45 68 92 52   

                                            cravings            price  \
0  Cuisine d'auteur,Cuisine du marché,Néobistrot,...   [De 16 à 35 €]   
1                Cuisine d'auteur,Italien,Néobistrot   [De 36 à 50 €]   
2                        Cuisine d'auteur,Néobistrot   [Plus de 51 €]   
3                        Cuisine d'auteur,Néobistrot   [De 16 à 35 €]   
4                        Cuisine d'auteur,Néobistrot   [Plus de 51 €]   
5             Kebabs,Sandwichs / Bagels,Lèche-doigts  [Moins de 15 €]   
6                         Bar à vins / Cave à manger   [De 16 à 35 €]   
7                         Bar à vins / Cave à manger   [De 16 à 35 €]   
8  Cuisine du marché,Méditerranéen,Sandwichs / Ba...   [De 16 à 35 €]   
9  Planches / Assiettes froides,Bar à vins / Cave...   [De 16 à 35 €]   

                        website                         address.position  \
0            www.badaboum.paris  [48.85345299999999, 2.3757510000000366]   
1           www.passerini.paris          [48.8494184, 2.374820600000021]   
2         www.atsushitanaka.com           [48.84974, 2.3544939999999315]   
3    www.chameleonrestaurant.fr   [48.86921700000001, 2.358015499999965]   
4  https://www.davidtoutain.com         [48.8602571, 2.3097285000000056]   
5                      barak.fr  [48.87551819999999, 2.3482715999999755]   
6             lemaryceleste.com           [48.861688, 2.364839599999982]   
7             www.early-june.fr   [48.87291459999999, 2.363441200000011]   
8            www.adar-paris.com          [48.8709479, 2.341645900000003]   
9               fulgurances.com          [48.8529714, 2.389381299999968]   

  address.postal_code             address.street_name_1  
0               75011           2, rue des Taillandiers  
1               75012               65, rue Traversière  
2               75005    4 bis, rue du Cardinal-Lemoine  
3               75010           70, rue René-Boulanger   
4               75007                    29 Rue Surcouf  
5               75010  62, rue du Faubourg-Poissonnière  
6               75003                   1, rue Commines  
7               75010            19, rue Jean Poulmarch  
8               75002         49, passage des Panoramas  
9               75011            5, rue Alexandre-Dumas

In [318]:
# on afiche le map en important folium 
import folium
from folium.plugins import MarkerCluster

arrondissement='75007'
spe="Cuisine d'auteur"

rest_map=folium.Map(location=[48.85,2.35],zoom_start=13, tiles='OpenStreetMap')
marker_cluster=folium.plugins.MarkerCluster().add_to(rest_map)

for i in df[(df['address.postal_code']== arrondissement) & (df['cravings'].str.contains(spe))][['name','address_position_lat','address_position_long']].index:
    marker=folium.Marker(location=(round(df['address_position_lat'].loc[i],6),round(df['address_position_long'].loc[i],6)),popup=(str(df['name'].loc[i]))).add_to(marker_cluster)
    
rest_map

In [311]:
#on crée une boucle qui sépare la latitude et longitude
lat=[]
long=[]
for elem in df['address.position']:
    lat.append(elem[0])
    long.append(elem[1])
    
df['address_position_lat']=lat
df['address_position_long']=long
    

In [312]:
#on vérifie si la latitude et longitude sont implantés 
df

id                      name           phone_num  \
0  5c8fba9a42719a5b5100003e    Restaurant du Badaboum   +33 1 85 09 75 46   
1  5a28125f42719a899e000001      Restaurant Passerini  + 33 1 43 42 27 56   
2  5a26861942719aec84000001          A.T. Restaurant   +33 1 56 81 94 08    
3  5a4fab5142719a3ba6000019      Chameleon Restaurant   +33 1 42 08 99 41   
4  5a297a7442719a8acf000007  Restaurant David Toutain   +33 1 45 50 11 10   
5  5c04f34742719ad8bf000370                     Barak   Pas de téléphone    
6  5a267bef42719a52bb00000f           Le Mary Celeste   +33 1 42 77 98 37   
7  5c62fbcf42719a9ac9000054                Early June        +33142854074   
8  5cec0f2542719aa67d0003ea                      Adar                       
9  5b87fc3942719a75a000006d      Fulgurances, en face   +33 1 45 68 92 52   

                                            cravings            price  \
0  Cuisine d'auteur,Cuisine du marché,Néobistrot,...   [De 16 à 35 €]   
1                Cuisine d'auteur,Italien,Néobistrot   [De 36 à 50 €]   
2                        Cuisine d'auteur,Néobistrot   [Plus de 51 €]   
3                        Cuisine d'auteur,Néobistrot   [De 16 à 35 €]   
4                        Cuisine d'auteur,Néobistrot   [Plus de 51 €]   
5             Kebabs,Sandwichs / Bagels,Lèche-doigts  [Moins de 15 €]   
6                         Bar à vins / Cave à manger   [De 16 à 35 €]   
7                         Bar à vins / Cave à manger   [De 16 à 35 €]   
8  Cuisine du marché,Méditerranéen,Sandwichs / Ba...   [De 16 à 35 €]   
9  Planches / Assiettes froides,Bar à vins / Cave...   [De 16 à 35 €]   

                        website                         address.position  \
0            www.badaboum.paris  [48.85345299999999, 2.3757510000000366]   
1           www.passerini.paris          [48.8494184, 2.374820600000021]   
2         www.atsushitanaka.com           [48.84974, 2.3544939999999315]   
3    www.chameleonrestaurant.fr   [48.86921700000001, 2.358015499999965]   
4  https://www.davidtoutain.com         [48.8602571, 2.3097285000000056]   
5                      barak.fr  [48.87551819999999, 2.3482715999999755]   
6             lemaryceleste.com           [48.861688, 2.364839599999982]   
7             www.early-june.fr   [48.87291459999999, 2.363441200000011]   
8            www.adar-paris.com          [48.8709479, 2.341645900000003]   
9               fulgurances.com          [48.8529714, 2.389381299999968]   

  address.postal_code             address.street_name_1  address_position_lat  \
0               75011           2, rue des Taillandiers             48.853453   
1               75012               65, rue Traversière             48.849418   
2               75005    4 bis, rue du Cardinal-Lemoine             48.849740   
3               75010           70, rue René-Boulanger              48.869217   
4               75007                    29 Rue Surcouf             48.860257   
5               75010  62, rue du Faubourg-Poissonnière             48.875518   
6               75003                   1, rue Commines             48.861688   
7               75010            19, rue Jean Poulmarch             48.872915   
8               75002         49, passage des Panoramas             48.870948   
9               75011            5, rue Alexandre-Dumas             48.852971   

   address_position_long  
0               2.375751  
1               2.374821  
2               2.354494  
3               2.358015  
4               2.309729  
5               2.348272  
6               2.364840  
7               2.363441  
8               2.341646  
9               2.389381